dev/CommonStyles/ProgressBar_themeresources.xaml

```diff
@@ -20,6 +21,8 @@
             <x:Double x:Key="ProgressBarIndicatorPauseOpacity">1</x:Double>
             <x:Double x:Key="ProgressBarThemeMinHeight">4</x:Double>
             <Thickness x:Key="ProgressBarBorderThemeThickness">1</Thickness>
+            <contract7NotPresent:Color x:Key="SystemErrorTextColor">#C50500</contract7NotPresent:Color>
```

Comment:
In the system, in high contrast this is #FFF000

Reply:
Updated it.